libs/components/tabs/src/lib/modules/tabs/tabset.component.scss

```diff
@@ -105,28 +181,11 @@ sky-tabset {
 
 .sky-tabset-btn-new,
 .sky-tabset-btn-open {
-  color: $sky-text-color-action-primary;
+  color: var(--sky-override-tabset-button-color, --sky-color-text-default);
 }
 
 .sky-theme-modern {
-  .sky-modal-content {
-    .sky-tabset {
-      --sky-tabset-margin: #{(-$sky-theme-modern-space-sm) (
-          -$sky-theme-modern-space-xl
-        ) $sky-theme-modern-space-xl};
-      padding-left: $sky-theme-modern-space-xl;
-      padding-right: $sky-theme-modern-space-xl;
-    }
-  }
-
   .sky-tabset {
-    border-bottom: 1px solid $sky-theme-modern-border-color-neutral-medium;
     overflow: visible; /* for drop shadow visibility on focused tabs */
```

Comment:
It looks like we already have an `overflow` specification for `sky-tabset` in the default styles - could we use an override of some sort to move this last style and eliminate the modern section all together?